README.md

```diff
@@ -163,11 +163,9 @@ with `B`.
   * `primary`: `true` | `false`, indicating whether this provider is the "main"
     provider responsible for recruiting a participant and performing physical
     measurements and biospecimen collection
-  * `organization`: Reference to an HPO organization, like `{"reference":
-    "Organization/PITT"}`
-  * `site`: array of
-    [references](http://hl7.org/fhir/references.html#reference) to an HPO site,
-    like `{"reference": "Organization/PITT_SOME_CLINIC"}`
+  * `awardee`: Reference to an awardee  pairing level, like `awardee: AZ_TUCSON`
```

Comment:
I think you can remove awardee: , organization: , and site: after the "like" here and below... those aren't a part of the value.

Also, you should mention UNSET as a possibility here.